libs/evm2wasm/CMakeLists.txt

```diff
@@ -15,5 +15,11 @@ if(BUILD_SHARED_LIBS)
   set_target_properties(libevm2wasm PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
 endif()
 
-target_include_directories(libevm2wasm PUBLIC ${include_dir})
-target_link_libraries(libevm2wasm PRIVATE binaryen::binaryen Threads::Threads fmt::fmt)
\ No newline at end of file
+target_include_directories(libevm2wasm PUBLIC $<BUILD_INTERFACE:${include_dir}>$<INSTALL_INTERFACE:include>)
```

Comment:
There's no space between the build/install_interface parts, is that intentional?

Reply:
This probably no strictly required but it is a common pattern I copy.

These are "generator expressions" and in build context this evaluates to `${include_dir}` (second part evaluates to nothing) and in install context this evaluates to `install` (first part evaluates to nothing).